app/routes.js

```diff
@@ -73,30 +74,65 @@ export const routes = (authProvider) => {
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
     const response = await postAddUser({ organisationId, data: req.body, accessToken });
     if (response.success) {
-      return res.redirect(`/organisations/${organisationId}/adduser/confirmation?userAdded=${response.userAdded}`);
+      req.userAdded = response.userAdded;
+      res.userAdded = response.userAdded;
+      return res.redirect(302, `/organisations/${organisationId}/adduser/confirmation?userAdded=${response.userAdded}`);
```

Comment:
does the 302 have to be specified here? noticed we don't this for other redirects

Reply:
no I was just trying to get it to work - will remove